src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -2309,7 +2309,7 @@ private void SetPeriodicEvaluation(WorkspaceModel ws)
             workspace.FromJsonGraphId = string.IsNullOrEmpty(filePath) ? WorkspaceModel.ComputeGraphIdFromJson(fileContents) : "";
             workspace.ScaleFactor = dynamoPreferences.ScaleFactor;
             
-            if (!IsTestMode)
+            if (!IsTestMode || !IsHeadless)
```

Comment:
Not sure why we have so many flags - isheadless, climode, etc. Wonder if we can do a cleanup to get rid of one. Making this PR to unblock Civil3D.

Reply:
I think there is a task in the backlog about cleaning this up.

Reply:
This should be (!IsTestMode && !IsHeadless).